launch_ground_station.bash

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+usage() {
+    echo "  options:"
+    echo "      -m: multi agent"
+    echo "      -t: launch keyboard teleoperation. Default not launch"
+    echo "      -v: open rviz. Default not launch"
+    echo "      -r: record rosbag. Default not launch"
+    echo "      -n: drone namespaces, comma separated. Default get from world config file"
```

Comment:
same comment, why to change it in two places instead on doing it directly in the config file once?

Reply:
What do you mean?

Reply:
On one hand, this file is generic, and there are platforms that do not have a configuration file where the namespaces to be used are set. On the other hand, just like before, you might want to control only one drone with the keyboard, record the rosbag of only one, etc. I think it doesn’t bother to have a parameter that the user does not need to modify by default.